### ISSUE TITLE
Hang Robot

### ISSUE THREAD

**Issue by IAmZeMuffinMan** [Bev-Facey-Robotics/BFH15254Repo#12]
Will want to be able to hang from the low bar at some point in the future, this is lower priority than having an functioning auto

**Comment by IAmZeMuffinMan** [comment]
Can confirm that this does in fact work, we need to iron out a few hiccups with the slides before we can reliably do it though.

**Comment by ream900** [comment]
One issue that I'm aware of is the strings get caught on where the supports were attached to the barrier on the pulleys. This could probably be easily fixed by editing the 3d model to have a barrier that has angled sides rather than vertical ones.

**Comment by IAmZeMuffinMan** [comment]
Has to retension the strings after hanging, however could be useful in a close match as a tie-breaker. Will use sparingly in meet, then will fix it for next comp

**Comment by IAmZeMuffinMan** [comment]
Need to test and maybe redo

**Comment by IAmZeMuffinMan** [comment]
Not a priority as of right now

**Comment by IAmZeMuffinMan** [comment]
So... it works really well for a lvl 2 ascent. We're seeing if we can get to a level 3 ascent for funsies.

**Comment by IAmZeMuffinMan** [comment]
Lvl 2 hang works very well. The lvl 3 ascent has been postponed until our auto works well
